app/com/arpnetworking/metrics/portal/reports/RecipientType.java

```diff
@@ -24,5 +24,15 @@
     /**
      * Email.
      */
-    EMAIL,
+    EMAIL("email");
+
+    public String getName() {
```

Comment:
An enum already has `public final String name()`, so this will be very confusing. Please be more clear.

Reply:
^ Or alternatively if these are the same then why do we need a lower case version?